SulimanJoya/To-Do-List-Project
# Testing to do list v2

## Changes
- Created a new branch(Testing-ToDo-List-v2)
- Use jest framework for testing
- Add checkbox-update, edit-description, and clear-completed functions in the update.js file
- Add test cases to check the functionality of:
- Clear-completed-task on the Local Storage
- Remove-task on the Local Storage
- Edit-description on the Local Storage

 


## Additional Info

## Checklist before requesting a review
- [X] Liner errors 
- [X] GitHub flow
- [X] Documented professionally 
- [X] Followed HTML & CSS best practices 
- [X] Followed JavaScript best practices
